lib/blacklight_cornell/virtual_browse.rb

```diff
@@ -154,6 +154,7 @@ def build_class_label(classlabel)
 
   # /get_previous
   def previous_callnumber
+    params.require([:callnum, :start])  # sometimes bots are calling this without required parameters - 400
```

Comment:
Nice - I'd like to see us be more explicit about what parameters are allowed and required in general more in controller actions (e.g. [strong parameters](https://guides.rubyonrails.org/action_controller_overview.html#strong-parameters). This is a great step in that direction.